examples/seismic/acoustic/operators.py

```diff
@@ -206,30 +225,38 @@ def GradientOperator(model, geometry, space_order=4, save=True,
         Option to store the entire (unrolled) wavefield.
     kernel : str, optional
         Type of discretization, centered or shifted.
+    factor : int, optional
+        Downsampling factor to save snapshots of the wavefield.
     """
     m = model.m
 
     # Gradient symbol and wavefield symbols
     grad = Function(name='grad', grid=model.grid)
-    u = TimeFunction(name='u', grid=model.grid, save=geometry.nt if save
-                     else None, time_order=2, space_order=space_order)
-    v = TimeFunction(name='v', grid=model.grid, save=None,
+    if factor: # Apply the imaging condition at the snapshots of the full wavefield
```

Comment:
Leave a blank line between the `grad = ` and this `if factor`

Reply:
Move the comment inside the body of the `if`

Reply:
Fixed